content/general/functions.textile

```diff
@@ -12,73 +12,61 @@ Generally, customers who want to receive events as they happen, use our "message
 
 h2(#messages). Messages
 
 If you are unfamiliar with how channels work in Ably, we recommend you first review the "realtime channel documentation":/realtime/channels-messages#channels.
 
 When a message is published on a channel, a Reactor Function message event is triggered, which in turn will send a message to your server-less function.
 
-Every request will contain your message within the @body@ part of the request. The contents of the @body@ depends on the format of your messages, and whether you've used an envelope or not.
-
-* No Envelope - Without an envelope, the contents of @body@ will simply be the contents message that was sent on the Ably Channel. 
-
-* Envelope - If you use an envelope, the @body@ will contain a bunch of extra attributes, which are the following:
-
+Every request will contain your message within the "HTTP message body":https://en.wikipedia.org/wiki/HTTP_message_body part of the request. The contents of the @HTTP message body@ depends on whether you've used an envelope or not. If you don't use an envelope, the @HTTP message body@ will only contain the @data@ payload from your message.
 
+@Enveloped@ - when the Reactor rule is configured to use an enveloper, the @HTTP message body@ will contain a bunch of extra attributes, which are the following:
 - source := the source of the request, namely "@channel.message"
-- appId := the ID of your application with Ably
-- channel := name of the channel the Ably Message was published on
+- appId := the "ID of your application":https://support.ably.io/support/solutions/articles/3000063083-how-do-i-find-my-app-id with Ably
+- channel := name of the channel the Ably @Message@ was published on
 - site := an internal site identifier indicating the data center from which the message was published
 - ruleId := an ID for the rule which has created this request
 - messages := an @Array@ of messages with fields for each message described below
 
-Each member of @messages@ has the following fields:
-
+Each member of @messages@ has the following attributes:
 - id := unique ID assigned by Ably to this message
-- name := a string representing the event name for the published message, see the "publish method":/rest/
+- name := a string representing the event name for the @message@
 - connectionId := the public unique identifier for the publisher's connection. Find out more about "@connectionId@":/realtime/connection#id
 - timestamp := the time in milliseconds since the epoch when this message was received by Ably
 - data := the data payload for the published message. Find out more about "message data":/rest/channels-messages#message
+More information on the @message@ type can be found "here":/realtime/versions/v0.8/channels-messages#message.
```

Comment:
Is this line displayed on the line after the definition list, or on the same line as the text part of the definition.  It would be a lot clearer to the reader of this textile what you intended if you either add a line break so that it's separate from the defintion list, or put it on the same line as above if an extension of that. Without looking at the rendering, I don't know.